task_5.2.py

```diff
@@ -0,0 +1,22 @@
+# Multi-paradigm programming languages, task №5.2
+# Andrey Fortus IKM-221a
+
+GENERAL_INFO = 'Multi-paradigm programming languages, task №5.2'
+STUDENT_INFO = 'Andrey Fortus IKM-221a variant №20'
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+
+# digits counter
+def digits_counter(n):
+    counter = 1
+    while abs(n) >= 10:
+        n = n / 10
```

Comment:
n /= 10

Reply:
your approach is fine.
consider a more efficient example:

counter = 0
while n:
    n //= 10
    counter += 1

Reply:
consider one more solution:

from itertools import takewhile, count, repeat

row = ((i, n) for i, n in enumerate(repeat(n)))
print(len(list(takewhile(lambda x: x[1] // (10 ** x[0]) > 0, row))))

Reply:
these two solutions do not work with negative numbers
![image_2023-03-19_18-44-10](https://user-images.githubusercontent.com/113716040/226191644-7cc7763f-2de3-47ea-8c8f-2056fc0eb7f7.png)
![image_2023-03-19_18-46-44](https://user-images.githubusercontent.com/113716040/226191661-e3b02151-3dfb-4a0d-8497-ce6ff187c1e7.png)

Reply:
you can apply abs() on the value